Extension/CHANGELOG.md

```diff
@@ -1,16 +1,20 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.9.6: March 18, 2022
+### Bug Fix
+* Redo 1.9.4 changes with a fix for issue [#9041](https://github.com/microsoft/vscode-cpptools/issues/9041)
+
 ## Version 1.9.5: March 17, 2022
-## Bug Fixes
+### Bug Fixes
 * Undo the 1.9.4 changes to fix the file corruption issue. [#9041](https://github.com/microsoft/vscode-cpptools/issues/9041).
```

Comment:
Incorrect trailing '.' after links

Reply:
Thanks.

Reply:
I added a trailing period of the 1.9.6 change because that's an usual case where it's referenced directly in a sentence.